src/components/html-text-input.js

```diff
@@ -8,23 +8,39 @@ import { Platform, TextInput, KeyboardAvoidingView } from 'react-native';
 import styles from './html-text-input.scss';
 
 // Gutenberg imports
-import { serialize } from '@wordpress/blocks';
+import { serialize, parse } from '@wordpress/blocks';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { withInstanceId, compose } from '@wordpress/compose';
 
 type PropsType = {
 	blocks: Array<Object>,
-	parseBlocksAction: string => mixed,
+	onChange: string => mixed,
+	onPersist: string => mixed,
+	html: string,
```

Comment:
A question:
Do we need `html` in both props and in the component state?
Seems redundant to have it in both unless I'm missing something important 🤔

Reply:
no you are right, removing it from props